lib/ruby2_keywords.rb

```diff
@@ -23,12 +23,9 @@ def ruby2_keywords
 end
 
 class Hash
-  unless respond_to?(:ruby2_keywords_hash?)
-    begin
-      $VERBOSE, verbose = nil, $VERBOSE
-      proc {|*a, **h| h}.call(
-    ensure
-      $VERBOSE = verbose
+  unless method_defined?(:ruby2_keywords_hash?)
```

Comment:
That's an incorrect check too BTW, there is `Hash.ruby2_keywords_hash?` and no `Hash#ruby2_keywords_hash?`.

Reply:
Nobu fixed it in https://github.com/ruby/ruby2_keywords/commit/51c47c060d9678ae2c28bcf415bc87346cba1860.